src/layouts/DialogLayout/DialogLayout.module.scss

```diff
@@ -0,0 +1,40 @@
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+|* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@use '../../styling/defs.scss' as bk;
+
+@layer baklava.components {
+  .bk-dialog-layout {
+    @include bk.component-base(bk-dialog-layout);
+
+    display: flex;
+    flex-direction: column;
+    gap: bk.$spacing-10;
+    padding-block-end: bk.$spacing-12;
+
+    .bk-dialog-layout__content {
+      display: flex;
+      flex-direction: row;
+      gap: bk.$spacing-12;
+    }
+
+    .bk-dialog-layout__content__aside {
+      inline-size: bk.$spacing-18;
+    }
+
+    hr {
+      border-block-end: bk.rem-from-px(1) solid bk.$theme-rule-default;
+      margin-block: bk.rem-from-px(70) bk.$spacing-2;
+    }
+
+    small {
+      color: bk.$theme-text-small-text-subtle;
+      font-size: bk.$font-size-s;
+    }
+
+    button {
+      font-size: bk.$font-size-s;
+    }
```

Comment:
These three CSS rules above are too generic. For example, this would style _any_ button in a dialog layout to have a small font.

- The `small` element should be moved into the `__aside` selector, and replaced with a class name, something like `.bk-dialog-layout__content__aside__hint`.
- For `hr`: you can probably just remove this entirely and give the `__hint` above a border-top.
- For `button`: I'm actually not sure what this is needed for. Maybe just remove it entirely?